### PR TITLE
Copy data & images mapped to words

### PR DESCRIPTION
**Pull request by jaydgoss**

### CODE REVIEW

**Review by hhff — APPROVED** [review]

File: app/lib/vudu.js [hunk 1]
```diff
@@ -20,152 +20,177 @@ const colors = {
...
+  display: 'initial',
+};
 
 c.mdFlex[breakpoints.md] = {
-  display: 'flex'
-}
+  display: 'flex',
+};
+
+c.relative = {
```
Comment (sepowitz) [inline app/lib/vudu.js:46]: these are already included with vudu

File: app/transforms/raw.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import DS from 'ember-data';
+
+export default DS.Transform.extend({
+  deserialize(serialized) {
+    return serialized;
+  },
+
+  serialize(deserialized) {
```
Comment (sepowitz) [inline app/transforms/raw.js:8]: do we need this if we aren't POSTing anything to contentful?

Comment (hhff) [reply app/transforms/raw.js:8]: we don't but it's a good practice to always implement hooks that utils are supposed to have

File: app/lib/vudu.js (hunk 1 above)
Comment (hhff) [reply app/lib/vudu.js:46]: @jaydgoss if you do `console.log(c)` at the end of this file you'll see all the built-in stuff

Comment (sepowitz) [reply app/lib/vudu.js:46]: I think I left a console.log in the application route, so it should appear in the console already

**Comment by hhff** [comment]
cool

**Comment by hhff** [comment]
yo also - with my update for ember (#31) Eslint is now included via the framework

**Pull request closed without merging**